README.md

```diff
@@ -135,7 +135,10 @@ Specifies RAID level. LVM RAID can be created as well.
 "Regular" RAID volume requires type to be `raid`.
 LVM RAID needs that volume has `storage_pools` parent with type `lvm`,
 `raid_disks` need to be specified as well.
-Accepted values are: `linear` (N/A for LVM RAID), `striped`, `raid0`, `raid1`, `raid4`, `raid5`, `raid6`, `raid10`
+Accepted values are:
+* for LVM RAID volume: `raid0`, `raid1`, `raid4`, `raid5`, `raid6`, `raid10`, `striped`, `mirror`
+* for RAID volume: `linear`, `raid0`, `raid1`, `raid4`, `raid5`, `raid6`, `raid10`
```

Comment:
Alternatively we could also hide the "special" levels (`linear` for MD and `striped`/`mirror` for LVM) to make the list simpler (but still supporting the levels when user specifies them).

Reply:
How about this?
```
Accepted values are:
* for LVM RAID volume: `raid0`, `raid1`, `raid4`, `raid5`, `raid6`, `raid10`, `striped`, `mirror`
* for RAID volume: `linear`, `raid0`, `raid1`, `raid4`, `raid5`, `raid6`, `raid10`

NOTE: `striped`, `mirror`, and `linear` are DEPRECATED and will be removed in the next major release.  Please use `raid0` instead of `striped`, `raid1` instead of `mirror`, and ??? instead of `linear`.
```
not sure what `linear` should be

Reply:
`linear` is an MD RAID level, I am not sure how widely used, but it is still supported. I don't think it makes sense to mark `mirror` and `striped` as deprecated -- these are old, but LVM doesn't plan to remove them.